src/SourceGenerators/Uno.UI.SourceGenerators/Helpers/SymbolExtensions.cs

```diff
@@ -91,7 +91,7 @@ public static IEnumerable<IEventSymbol> GetEvents(INamedTypeSymbol? symbol)
 		/// </summary>
 		/// <param name="symbol">The current symbol</param>
 		/// <param name="typeName">A potential base class.</param>
-		public static bool Is(this INamedTypeSymbol? symbol, string typeName)
+		public static bool InheritsFrom(this INamedTypeSymbol? symbol, string typeName)
```

Comment:
I have not problem with this change, as it makes it more readable, but it'll be better if you make it a different PR.

Reply:
@jeromelaban Will do. I'll revert the change and have it in a new PR. Going to squash that along with proper testing. Would be helpful if you or someone can guide me to the proper way to testing this.

Reply:
I don't think you'll be able to make a unit test out of this because of platform dependencies, though you can probably make one [in here](https://github.com/unoplatform/uno/tree/master/src/SourceGenerators/XamlGenerationTests), as it is built to validate that the generated code can compile, or make add it to the [runtime tests](https://github.com/unoplatform/uno/tree/master/src/Uno.UI.RuntimeTests) assembly, even if it's not really testing anything at runtime.

Reply:
@jeromelaban The first link is for `XamlGenerationTests`, does it support native ctor generation testing too?

Reply:
It runs all the generators, so yes, it will validate this as well. But after reflection, it's best if you add it to the runtime tests.